lib/Constants.vala

```diff
@@ -29,6 +29,7 @@ namespace Gala {
         // Duration of the snap animation as used by maximize/unmaximize
         SNAP = 250,
         // Duration of the workspace switch animation
-        WORKSPACE_SWITCH = 300,
+        WORKSPACE_SWITCH_MIN = 100,
```

Comment:
This feels like 10000% better than master already, but I would be okay with going even longer since this is such a big animation. Even at `300` for minimum, it still feels pretty fast imo